client/blocks/login/social.jsx

```diff
@@ -182,12 +190,21 @@ class SocialLoginForm extends Component {
 						responseHandler={ this.handleGoogleResponse }
 						uxMode={ uxMode }
 						redirectUri={ this.getRedirectUrl( uxMode, 'google' ) }
-						onClick={ this.trackLogin.bind( null, 'google' ) }
+						onClick={ this.trackLoginAndRememberRedirect.bind( null, 'google' ) }
+						socialServiceResponse={
+							this.props.socialService === 'google' ? this.props.socialServiceResponse : null
+						}
 					/>
 
 					<AppleLoginButton
+						clientId={ config( 'apple_oauth_client_id' ) }
 						responseHandler={ this.handleAppleResponse }
-						onClick={ this.trackLogin.bind( null, 'apple' ) }
+						uxMode={ uxModeApple }
+						redirectUri={ this.getRedirectUrl( uxMode, 'apple' ) }
```

Comment:
If the `uxMode` parameter really is needed, make sure you really want to pass `uxMode` and not `uxModeApple` here.

Reply:
Indeed `uxMode` is not needed anymore!